processor/attributesprocessor/attributes_test.go

```diff
@@ -867,6 +868,27 @@ func TestAttributes_Matching_False(t *testing.T) {
 				Attributes: []matchAttribute{},
 			},
 		},
+
+		{
+			name: "span name doesn't match",
```

Comment:
@tigrannajaryan per conversation on other PR, please add the underscores :)

Reply:
Done.